src/store/reducers/rootReducer.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
+import { appReducer } from './appReducer.js';
```

Comment:
Better to write like this `import { appReducer as app } from './appReducer.js'` for better understanding state